x-pack/plugins/uptime/public/components/queries/filter_bar/filter_bar.tsx

```diff
@@ -126,8 +126,8 @@ export const FilterBar = ({
         {
           type: 'field_value_selection',
           field: 'monitor.type',
-          name: i18n.translate('xpack.uptime.filterBar.options.typeLabel', {
-            defaultMessage: 'Type',
+          name: i18n.translate('xpack.uptime.filterBar.options.schemeLabel', {
+            defaultMessage: 'Scheme',
```

Comment:
++ - this is only called `Type` currently because the name of the field we were using changed between `6.x` and `7.x`.